src/app/pages/ArticlePage/index.jsx

```diff
@@ -42,14 +59,62 @@ const StyledMain = styled.main`
   flex-grow: 1;
 `;
 
+const MostReadSection = styled.section.attrs(() => ({
+  role: 'region',
+  'aria-labelledby': 'Most-Read',
+  'data-e2e': 'most-read',
+}))``;
+
+const ConstrainedMostReadSection = styled(MostReadSection)`
```

Comment:
Should we call this constant `ArticleMostReadSection` as we do with the Front Page https://github.com/bbc/simorgh/pull/6175/files ?

Reply:
yes, that makes more sense